DemoUI/css/headers/index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
```

Comment:
Basically, we can consider just removing any HTML files in the CSS directory. These might have come from `bower install`s though, so maybe we just don't touch them.